packages/fxa-settings/src/components/Settings/PageSettings/index.tsx

```diff
@@ -31,6 +34,26 @@ export const PageSettings = (_: RouteComponentProps) => {
     GleanMetrics.accountPref.view();
   }, []);
 
+  const accountRecoveryNotificationProps = {
+    headerImage: keyImage,
+    ctaText: ftlMsgResolver.getMsg(
+      'account-recovery-notification-cta',
+      'Create'
+    ),
+    headerValue: ftlMsgResolver.getMsg(
+      'account-recovery-notification-header-value',
+      "Don't lose your data if you forget your password"
+    ),
+    headerDescription: ftlMsgResolver.getMsg(
+      'account-recovery-notification-header-description',
+      'Create an Account Recovery Key to restore your sync browsing data if you ever forget your password.'
```

Comment:
I see this is the same copy as in Figma, but I wonder if this should be `account recovery key` lowercase. 🤔